src/components/Catalog.vue

```diff
@@ -250,13 +294,69 @@ export default {
     };
   },
   asyncComputed: {
+    collections: {
+      default: [],
+      lazy: true,
+      async get() {
+        const externalCollections = this.links.find(x => x.rel === "data");
+        if (externalCollections === undefined) {
+          return [];
+        }
+
+        try {
+          const rsp = await fetch(externalCollections.href);
+          if (!rsp.ok) {
+            console.warn(await rsp.text());
+            return [];
+          }
+
+          const data = await rsp.json();
+          if (!data || !Array.isArray(data.collections)) {
+            console.warn(await rsp.text());
+            return [];
+          }
+
+          return data.collections
+            .map(collection => {
+              // strip /collection from the target path
+              let p = this.path.replace(/^\/collection/, "");
+              if (!p.endsWith("/")) {
+                p += "/";
+              }
+
+              // Try to get the location of the collection
+              let href = externalCollections.href + '/collections/' + collection.id;
+              if (Array.isArray(collection.links)) {
+                let selfLink = collection.links.find(l => l.rel == 'self');
+                if (selfLink && selfLink.href) {
+                  href = selfLink.href;
+                }
+              }
+
+              const slug = this.slugify(this.resolve(href, this.url));
+              const to = [p, slug].join("");
+
+              return Object.assign(collection, {
+                path: href,
+                to,
+                title: collection.title || collection.id || href,
+                url: this.resolve(href, this.url)
```

Comment:
Consider assigning `this.resolve(href, this.url)` to a const to use once (also used in definition of slug).

Reply:
Also just copied, this time from https://github.com/radiantearth/stac-browser/pull/59/files#diff-fbae1f7e04ad34c70ca90231db3dd296764917ad88929a0860f4fe50e8627320L354 - So I guess I should also change it there?